combirepo/directory_downloader.py

```diff
@@ -186,16 +186,18 @@ def mkdir():
             name = link.rsplit('/', 1)[1]
             if '?' in name:
                 continue
-            links_resolved.append(link)
-            if packages_list is not None:
-                base_name = splitFilename(os.path.basename(name))
-                if base_name:
-                    rpm_name = base_name[0]
-                    if rpm_name in packages_list:
-                        names.append(name)
+            if name.endswith('.rpm'):
```

Comment:
What else can be provided in packages list?
Do we really need to filter out rpms?

Reply:
Here html is processed too.